lib/nermesterts/game_picker.ex

```diff
@@ -26,6 +26,8 @@ defmodule GamePicker do
       nil
   """
   def pick_game(games_list, num_players) do
+    :rand.seed(:exsplus, DateTime.utc_now |> DateTime.to_date |> Date.to_erl)
```

Comment:
So, this and the call to `Enum.take_random` are non-deterministic...
That said, if you want to test this and allow for this to be deterministic, maybe a "game randomization" function would be in order - something that you could inject.

There's a number of ways to inject it, but what this *could* look like:
```elixir
def pick_game(games_list, num_players, randomizer \\ GamePicker.Randomizer) do 
...
|> randomizer.randomize()
end
```
You could then test the injected piece independently if you want. 
Just a thought.